python/vision_explanation_methods/explanations/drise.py

```diff
@@ -127,7 +127,8 @@ def compute_affinity_scores(
     :rtype: Tensor of shape D, where D is number of base detections
     """
     score_matrix = compute_affinity_matrix(base_detections, masked_detections)
-
+    if not (score_matrix.numel()):
```

Comment:
are there any tests that could be added with these fixes?

Reply:
That's a great idea! However, in the process of writing tests for this, I actually realized that this fix wasn't in the best location, and it would be better to have error handling in the responsibleai_vision side. This is because this version of the code is difficult to verify, and could lead to some unexpected behavior.